frontend/packages/ceph-storage-plugin/src/utils/storage-class-params.ts

```diff
@@ -0,0 +1,5 @@
+import { StorageClassFormProvisoners } from '../components/ocs-storage-class-form';
+
+export const getStorageClassProvisoner = () => {
+  return StorageClassFormProvisoners;
+};
```

Comment:
Why doing this? why not just import `StorageClassFormProvisoners` where ever required.

Reply:
+1